runtime/src/bank/accounts_lt_hash.rs

```diff
@@ -325,6 +332,22 @@ impl Bank {
             return;
         }
 
+        if self
+            .freeze_started_for_accounts_lt_hash
+            .load(Ordering::Acquire)
+        {
```

Comment:
As discussed offline, checking here isn't sufficient to guarantee freeze hasn't started before we modify the lt hash cache.